Return to parent work view after edit a file

Xiaoming opened the pull request.

Xiaoming committed:
HYDRASIR-307: return to work view after edit a file

jeremyf commented:
I'm reviewing the failed build it initially does not appear to be related to what you've done.

jeremyf committed:
Tidying up file update feature behavior

When we successfully edit a generic file, we are returning to the
parent object. The feature was not reflecting this behavior.

HYDRASIR-307 #close

JIRA References:
https://jira.duraspace.org/browse/HYDRASIR-307
